sessions/react-global-state/demo-end/pages/index.js

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+const Container = styled.div`
+  display: grid;
+  gap: 4rem;
+  padding-inline-start: 2rem;
+`;
+
+const Title = styled.h1`
+  font-size: 3rem;
+  padding: 1rem;
+`;
+
+const Flex = styled.div`
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const StyledLink = styled(Link)`
+  font-size: 2rem;
+  text-decoration: none;
+  color: #fff;
+  background: var(--primary-color);
+  width: 50%;
+  padding-inline-start: 0.5rem;
+`;
+
+export default function HomePage({ countSum, dragonCount }) {
```

Comment:
The homepage looks very different from the overview page. Maybe we can find a way to give them a similar appeal. 

Ideas:
- use header and footer on both pages
- use title to both pages

The `AppLayout` component could be wrapped around both pages in the `_app.js`